provider/fs/src/export/fs_exporter.rs

```diff
@@ -73,13 +73,7 @@ impl DataExporter<SerializeMarker> for FilesystemExporter {
         obj: DataPayload<SerializeMarker>,
     ) -> Result<(), DataError> {
         let mut path_buf = self.root.clone();
-        path_buf.extend(req.resource_path.key.iter_components());
-        path_buf.extend(
-            req.resource_path
-                .options
-                .iter_components()
-                .map(|s| PathBuf::from(&*s)),
-        );
+        path_buf.push(req.resource_path.to_string());
```

Comment:
```suggestion
        path_buf.push(req.resource_path.writeable_to_string());
```

Reply:
Ah yeah I didn't do this because this crate doesn't have the `writeable` dependency yet. Should I add it?

Reply:
This is an owned cow anyway, as the path string consists of both key and options.